ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -225,6 +233,15 @@ export type wireframeRectangle = wireframeBase & {
     type: 'rectangle'
 }
 
+export type wireframeWebView = wireframeBase & {
+    type: 'web_view'
```

Comment:
@pauldambra 
Maybe we add a `label` here as well so I can set the URL as a label? or add a new field?
Right now the label defaults to `web_view` because the `wireframeWebView` rendering just forwards to the `wireframePlaceholder` rendering.